README.md

```diff
@@ -53,24 +55,31 @@ git pull
 
 ## Configuration
 
-To use this module, add it to the modules array in the `config/config.js` file.
+To use this module, add a configuration to the modules array in the `config/config.js` file.
+
+*Note*: Some of the module names used in the following examples are fictitious.
```

Comment:
I would suggest making it explicit that there are two ways to configure the module. Maybe even suggesting one for folks that don't know which to use

Reply:
I added a explanation that there are two ways to configure. Please check if that's okay :slightly_smiling_face: